server/sonar-web/src/main/js/main/nav/global/search-view.js

```diff
@@ -184,7 +184,7 @@ export default Marionette.LayoutView.extend({
     });
     const qualifiers = this.model.get('qualifiers').map(function (q, index) {
       return {
-        url: '/all_projects?qualifier=' + encodeURIComponent(q),
+        url: window.baseUrl + '/all_projects?qualifier=' + encodeURIComponent(q),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)